update.php

```diff
@@ -137,6 +137,19 @@
 			'download_url' => 'https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar',
 		),
 	),
+	'7.4' => array(
+		'php' => array(
+			'base_name'       => 'php:7.4-rc-fpm',
+			'gd'              => true,
+			'extensions'      => array( 'gd', 'opcache', 'mysqli', 'zip' ),
+			'pecl_extensions' => array( 'xdebug-2.7.2' ),
```

Comment:
```suggestion
			'pecl_extensions' => array( 'xdebug-2.8.0beta1' ),
```

Xdebug 2.8.0beta1 is required for PHP 7.4 support.